pkgs/development/compilers/ghc/7.0.4.nix

```diff
@@ -13,6 +13,8 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ ghc perl gmp ncurses ];
 
+  enableParallelBuilding = true;
```

Comment:
I am worried that enabling parallelism in older expression will cause trouble. We had parallel builds enabled in the past, and it did us no good: cb21683794269846ba76e66a9c3b1869ad4b817d.